catroid/src/androidTest/java/org/catrobat/catroid/uiespresso/ui/dialog/RegularExpressionAssistantDialogTest.java

```diff
@@ -146,4 +146,13 @@ public void testDoesHtmlExtractorOpensCorrectDialog() {
 		onView(withText(R.string.formula_editor_regex_html_extractor_dialog_title)).perform(click()); //click on HTML Extractor
 		onView(withText(R.string.formula_editor_regex_html_extractor_dialog_title)).check(matches(isDisplayed()));
 	}
+
+	@Test
+	public void testDoesJsonExtractorOpensCorrectDialog() {
+		clickOnAssistantInFunctionList();
+
+		onView(withText(R.string.formula_editor_function_regex_json_extractor_title)).perform(click()); //click on HTML
```

Comment:
```suggestion
		onView(withText(R.string.formula_editor_function_regex_json_extractor_title)).perform(click());
```
Why HTML here? 

Please leave all unnecessary comments away. Also delete the same one a few lines higher up where this was copied from. This is a textbook example ;-)  where a copy-paste led to a comment that is incorrect. Nobody will look at it anymore, and it will remain forever in the code. In a few years some novice developer will look at it and "correct" it from JSON to HTML... --- hopefully not.

Reply:
Done.